### DISCUSSION TITLE
Cannot install trilium as ./configure shows no input file.

### DISCUSSION THREAD

**Discussion by aditya-agarkar-2021** [TriliumNext/Trilium#6317] [Q&A] [answered]
For bug reports, **PLEASE mention version of Trilium you're using** and also include **log files** from following location:

* `/home/[user]/.local/share/trilium-data/log` for Linux
* `C:\Users\[user]\AppData\Roaming\trilium-data\log` for Windows Vista and up
* `/Users/[user]/Library/Application Support/trilium-data/log` for Mac OS

**Comment by zadam** [comment]
What `./configure`?

**Comment by aditya-agarkar-2021** [comment]
Hi, I downloaded the tar.xz file for linux and extracted it. But couldn't pass the make command as it showed there was no configure,ac or configure.in file. 
I am very new to Linux and Ubuntu, so please guide me.

**Comment by zadam** [comment, marked as answer]
This is a step by step guide: https://github.com/zadam/trilium/wiki/Packaged-server-installation
